Scripts/Python/grsnWallPython.py

```diff
@@ -404,8 +407,9 @@ def OnNotify(self,state,id,events):
                     SouthPanelSound.run(self.key, state='denied')
                 else:
                     self.ChangeGameState(kSouth, kWait)
-                    SouthTubeOpen.run(self.key)
                     SouthPanelSound.run(self.key, state='gameStart')
+                    if not bool(PtGetPlayerList()):
```

Comment:
```suggestion
                    if not PtGetPlayerList():
```

Reply:
This check is for the Solo Mode wall game.
PtGetPlayerList() returns a list of players, without the bool conversion it wont properly work.

Reply:
Empty list is False-y and populated list is True-thy, so Hoikas is correct that the cast is not needed. Tested in python REPL:
```
>>> foo = []
>>> if not foo:
...     print("x")
...
x
>>> foo = [1]
>>> if not foo:
...     print("x")
...
>>>
```